packages/sdk/src/protocols/light_push.ts

```diff
@@ -90,18 +90,20 @@ class LightPushSDK extends BaseProtocolSDK implements ILightPushSDK {
         }
         if (failure) {
           if (failure.peerId) {
-            try {
-              await this.renewPeer(failure.peerId);
-              log.info("Renewed peer", failure.peerId.toString());
-            } catch (error) {
-              log.error("Failed to renew peer", error);
-            }
+            // schedule renewal as we don't need to wait for it
+            this.renewPeer(failure.peerId)
+              .then(() => {
+                log.info("Renewed peer", failure?.peerId?.toString());
+              })
+              .catch((error) => {
+                log.error("Failed to renew peer", error);
+              });
           }
 
           failures.push(failure);
         }
       } else {
-        log.error("Failed to send message to peer", result.reason);
+        log.error("Promise rejected while sending:", result.reason);
```

Comment:
added it because this branch of code should not be executed and if we land here - it means something unexpected happened and promise failed

if you don't like `promise` word - I will name it `Failed unexpectedly while sending`